src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -3012,6 +3012,11 @@ public int forceKeyspaceCleanup(String keyspaceName, String... columnFamilies) t
         return forceKeyspaceCleanup(0, keyspaceName, columnFamilies);
     }
 
+    public void createCleanupEntryForCfIfNotExists(String keyspaceName, String columnFamily, Optional<Instant> cleanupTs)
+    {
+        cleanupState.createCleanupEntryForCfIfNotExists(keyspaceName, columnFamily, cleanupTs);
+    }
```

Comment:
What's the advantage of pulling this into its own function instead of directly modifying the `cleanupState.createCleanupEntryForTableIfNotExists(...)` call [here](https://github.com/palantir/cassandra/pull/371/files#diff-9bf2c26bc294ef9085e16bf287490223665eaa2eb8ec24bcf5bd8653c713644bL3029)?

Reply:
Because this function is called from `Keyspace` and cleanupState is a private field

Reply:
Makes sense - it's less invasive than modifying the existing function, so this change makes sense!